src/routes/index.js

```diff
@@ -1,7 +1,11 @@
 const router = require('express').Router();
 const userRoutes = require("./v1/user.routes");
 const unidadesRoutes = require('./v1/unidades');
+const geracaoRoutes = require('./v1/geracao.routes');
 
+router.use(unidadesRoutes);
+router.use(UserRoutes);
+router.use(geracaoRoutes);
 router.use([unidadesRoutes, userRoutes]);
```

Comment:
Seria melhor agrupar todas as rotas em uma matriz
```suggestion
router.use([unidadesRoutes, userRoutes, unidadesRoutes, geracaoRoutes]);
```